packages/twenty-server/src/engine/core-modules/billing/services/billing-subscription.service.ts

```diff
@@ -148,6 +97,23 @@ export class BillingSubscriptionService {
     }
   }
 
+  async getWorkspaceEntitlementByKey(
+    workspaceId: string,
+    lookupKey: FeatureStripeLookupKey,
+  ) {
+    const entitlement = await this.billingEntitlementRepository.findOneBy({
+      workspaceId,
+      key: lookupKey,
+      value: true,
+    });
+
+    if (!entitlement?.value) {
```

Comment:
split to be fully explicit
if (entitlement === undefined) { return false; }
return entitlement.value